### ISSUE TITLE
  model.add(Merge([model1, model2], mode='concat'))

### ISSUE THREAD

**Issue by prachinarang24** [Currie32/Predicting-the-Dow-Jones-with-Headlines#3]
This is throwing error please help as I am new to programming in deep learning

**Comment by rajatsharma369007** [comment]
Same issue here, Merge layer has been deprecated from the library. can you provide some alternative to this layer.

**Comment by rajatsharma369007** [comment]
solved:

    # model = Sequential()

    model = Add()([model1.output, model2.output])
    model = Dense(hidden_dims, kernel_initializer=weights)(model)
    model = Dropout(dropout)(model)
    if deeper == True:
        model = Dense(hidden_dims//2, kernel_initializer=weights)(model)
        model = Dropout(dropout)(model)
    model = Dense(1, kernel_initializer = weights, name='output')(model)
    new_model = Model([model1.input, model2.input], model)
    new_model.compile(loss='mean_squared_error', optimizer=Adam(lr=learning_rate, clipvalue=1.0))
    return new_model

import Add layer in the top

**Comment by gnaneshnammi** [comment]
How to import Add layer

**Comment by asaxenastanford** [comment]
@gnaneshnammi 
from keras.layers import *
from keras.layers import add
